.github/workflows/ci.yaml

```diff
@@ -0,0 +1,50 @@
+name: CI Testing
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
How about
```yaml
jobs:
  build:
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ ubuntu-latest, windows-latest, macos-latest ]
        python-version: ['3.6', '3.7', '3.8', '3.9']
    if: ${{ matrix.os == 'ubuntu-latest' || matrix.python-version == '3.9' }}
```

Reply:
Also, the CI tools seem to handle it ok, but it's a good idea to quote the version numbers since they get interpreted as floats otherwise.

Reply:
```
if: ${{ matrix.os == 'ubuntu-latest' || matrix.python-version == '3.9' }}
```

Not sure why but the above seems not to work:

```
Unrecognized named-value: 'matrix'
```